src/mempool_limit.h

```diff
@@ -57,80 +59,62 @@ class RecentlyEvictedList
 
 // The mempool of a node holds a set of transactions. Each transaction has a *cost*,
 // which is an integer defined as:
-//   max(serialized transaction size in bytes, 4000)
+//   max(memory usage in bytes, 4000)
+//
 // Each transaction also has an *eviction weight*, which is *cost* + *fee_penalty*,
-// where *fee_penalty* is 16000 if the transaction pays a fee less than 10000 zatoshi,
-// otherwise 0.
-struct TxWeight {
-    int64_t cost;
-    int64_t evictionWeight; // *cost* + *fee_penalty*
-
-    TxWeight(int64_t cost_, int64_t evictionWeight_)
-        : cost(cost_), evictionWeight(evictionWeight_) {}
-
-    TxWeight add(const TxWeight& other) const;
-    TxWeight negate() const;
-};
+// where *fee_penalty* is 16000 if the transaction pays a fee less than the
+// ZIP 317 conventional fee, otherwise 0.
```

Comment:
This PR doesn't bring in the ZIP 317 changes, right? So technically this comment should be left as-is. Very weakly blocking.

Reply:
```suggestion
// where *fee_penalty* is 16000 if the transaction pays a fee less than the
// conventional fee, otherwise 0.
```